Algorithmia/AlgorithmiaClient.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  AlgorithmiaClient.swift
+//  AlgorithmiaSwiftDemo
+//
+//  Created by Erik Ilyin on 11/1/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import Foundation
+
+class AlgorithmiaClient {
+    var apiClient:AlgoAPIClient
+    init() {
+        self.apiClient = AlgoAPIClient(auth: nil)
```

Comment:
I think we need a field like `self.baseApiAddress` here. The constructor should take a baseApiAddress. If it is not null, set it to the value passed in. If not, use the environment variable. Lastly, we can default to "https://api.algorithmia.com".

This makes it easier for us to use this client to hit our non-prod environments. For an example of this, look at https://github.com/algorithmiaio/algorithmia-r/blob/master/R/Client.r#L4-L10